File: CS/08_Background/SetMasterBackground.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Presentation;

namespace SetMasterBackground
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create a PPT document
            Presentation presentation = new Presentation();

            //Set the slide background of master
            presentation.Masters[0].SlideBackground.Type = Spire.Presentation.Drawing.BackgroundType.Custom;
            presentation.Masters[0].SlideBackground.Fill.FillType = Spire.Presentation.Drawing.FillFormatType.Solid;
            presentation.Masters[0].SlideBackground.Fill.SolidColor.Color = Color.LightSalmon;

            //Save the document
            string result = "SetSlideMasterBackground_result.pptx";
            presentation.SaveToFile(result, FileFormat.Pptx2013);

            //Launch the file
            OutputViewer(result);
        }
        private void OutputViewer(string filename)
        {
            try
            {
                System.Diagnostics.Process.Start(filename);
            }
            catch { }
        }
    }
}

File: VB/08_Background/SetMasterBackground.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Presentation

Namespace SetMasterBackground
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create a PPT document
			Dim presentation As New Presentation()

			'Set the slide background of master
			presentation.Masters(0).SlideBackground.Type = Spire.Presentation.Drawing.BackgroundType.Custom
			presentation.Masters(0).SlideBackground.Fill.FillType = Spire.Presentation.Drawing.FillFormatType.Solid
			presentation.Masters(0).SlideBackground.Fill.SolidColor.Color = Color.LightSalmon

			'Save the document
			Dim result As String = "SetSlideMasterBackground_result.pptx"
			presentation.SaveToFile(result, FileFormat.Pptx2013)

			'Launch the file
			OutputViewer(result)
		End Sub
		Private Sub OutputViewer(ByVal filename As String)
			Try
				Process.Start(filename)
			Catch
			End Try
		End Sub
	End Class
End Namespace